src/api/clients/IOClients/infra/VBase.ts

```diff
@@ -21,10 +22,30 @@ export class VBase extends InfraClient {
     })
   }
 
+  public resolveConflict = (bucketKey: string, path: string, content: any) => {
```

Comment:
Nit: WDYT about adding jsdoc annotation in these new functions? Since they are used in a outside context, it might be helpful to add some shared comments. I think it's important to mention they are very specific for pages-graphql stuff.

Reply:
Agree! But I wonder if it's only used for pages-graphql.

Reply:
I think it's safe to assume that since the bucket name has pages-graphql hard coded in the path.